codes/quantum/qudits/non_stablizer.yml

```diff
@@ -0,0 +1,38 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: non_stablizer
+physical: qudits
+logical: qudits
+
+name: 'Non-stablizer code'
+introduced: '\cite{arxiv:quant-ph/0210097}'
+
+
+description: |
+    Non-stablizer codes are natural consequence of the QECC formulation based on Weyl relations and Fourier descriptions \cite{arxiv:quant-ph/0210097}.
+    Asymptotically good non-stablizer codes admitting efficient encoding and decoding procedures have been constructed.
+
+
+
+features:
+
+  encoders:
+    - 'The encoding circuit involves the application of quantum Fourier transformation. (If the underlying registers are qubits, this recudes to a single Hadamard gate.)'
+
+  decoders:
+    - 'The decoding circuit involves the application of phase estimation.'
+
+
+
+notes:
+  - 'For any \(\alpha\)-good \(n\times n\) matrix over \(\mathbb{F}_2\) and \(\lfloor\alpha n\rfloor\)-pure maximal Gottesman subgroup of the error group \(\mathcal{E}, 0<\alpha<1\), there exists an \([n,\frac{2^n}{N(n,2,\lfloor\alpha n\rfloor)},2\alpha n]\) QECC, where \(N(n,q,d)=\sum_{i=1}^d\begin{pmatrix} n \\ i\end{pmatrix}(q^2-1)^i\). (For definitions, see \cite{arxiv:quant-ph/0210097})'
```

Comment:
Use \mathbb{Z} since field properties aren't too important here.

Reply:
Would be good to try and define Gottesman group here. Also, this first note should go in protection.